.github/workflows/Tests.yaml

```diff
@@ -163,7 +163,14 @@ jobs:
   doc_deploy:
     name: Deploy documentation
     needs: doc_check
-    if: github.event_name == 'push' && github.ref == 'refs/heads/develop'
+    # Enabling documentation deployment only on pushes to
+    # sxs-collaboration/spectre for now, since the action requires a personal
+    # access token until the issue linked below is resolved. Once the action
+    # can use the standard `GITHUB_TOKEN` we can enable this job also on forks.
+    if: >-
+      github.event_name == 'push'
```

Comment:
Would it be better to check that the GH_PAGES_DEPLOY_TOKEN is set? Then one can run this on the branch if desired. Note this is just a comment for discussion, not a binding request :)

Reply:
This can’t be done within the `if` clause at the moment, but only in the script of a step. So the job would always run and appear as successful, even if it didn’t do anything. Since we only need this check until the GH Actions issue with gh-pages and the `GITHUB_TOKEN` is resolved I suggest to disable on forks until then.

Reply:
Okay, sounds good. Thanks!